Session1.Rmd

```diff
@@ -538,36 +576,36 @@ head(gapminder_gdp)
 
```

Comment:
Not sure if you already do this and if there is time, but might be nice to briefly show the difference between the outcome of summarize and the outcome of mutate. For instance, both can calculate the mean, but the resulting dataframe is different.